docs/content/protondrive.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Proton Drive"
+description: "Rclone docs for Proton Drive"
+versionIntroduced: "v1.64.0"
+---
+
+# {{< icon "fa fa-atom-simple" >}} Proton Drive
+
+[Proton Drive](https://proton.me/drive) is an end-to-end encrypted Swiss vault for your files that protects your data.
+
+This is a rclone backend for Proton Drive which supports the file transfer
+features of Proton Drive using the same client-side encryption.
+
+Due to the fact that Proton Drive doesn't publish its API documentation, this backend is implemented with the best effort by reading the open-sourced client source code and observing the Proton Drive traffic on the browser.
```

Comment:
We normally hard wrap the docs to 80ish chars too.

Reply:
Got it! Fixed as requested.